doc/user-guide/io.rst

```diff
@@ -984,6 +984,42 @@ reads. Because this fall-back option is so much slower, xarray issues a
     3. Explicitly setting ``consolidated=True``, to raise an error in this case
        instead of falling back to try reading non-consolidated metadata.
 
+.. _io.kerchunk:
+
+Kerchunk
+--------
+
+`Kerchunk <https://fsspec.github.io/kerchunk/index.html>`_ is a Python library that allows you to access chunked and compressed data formats (such as NetCDF3, NetCDF4, HDF5, GRIB2, TIFF & FITS), many of which are primary data formats for many data archives, as if they were in ARCO formats such as `Zarr`_ which allows for parallel, chunk-specific access. Instead of creating a new copy of the dataset in the Zarr spec/format, Kerchunk reads through the data archive and extracts the byte range and compression information of each chunk, then writes that information to a ``.json`` file, which is later used as an ephemeral ``zarr`` dataset.
```

Comment:
> "zarr" is the only backend used by engine="kerchunk"

But "kerchunk" is the backend engine, right, ` xr.open_dataset(d, engine="kerchunk")`?

Reply:
Yes, engine="kerchunk".

My comment is in response to
> as if they were in ARCO formats such as `Zarr`

where "such as" is unnecessary, since we are always interfacing via zarr.